app/src/js/components/Rules/rule.js

```diff
@@ -169,26 +190,29 @@ class Rule extends React.Component {
                 state: {
                   name: ruleName
                 }
-              }}>Copy</Link>
+              }}>Copy Rule</Link>
             <Link
               className='button button--edit button--small button--green form-group__element--right'
-              to={`/rules/edit/${ruleName}`}>Edit</Link>
+              to={`/rules/edit/${ruleName}`}>Edit Rule</Link>
             {lastUpdated(data.timestamp)}
-            {data.state ? (
-              <dl className='status--process'>
-                <dt>State:</dt>
-                <dd className={data.state.toLowerCase()}>{displayCase(data.state)}</dd>
-              </dl>
-            ) : null}
-
           </div>
         </section>
         <section className='page__section'>
           {errors.length ? <ErrorReport report={errors} /> : null}
           <div className='heading__wrapper--border'>
             <h2 className='heading--medium with-description'>Rule Overview</h2>
           </div>
-          <Metadata data={data} accessors={metaAccessors} />
+          <div className="rule__state">
+            {data.state ? (
+              <dl className='status--process'>
+                <dt>State:</dt>
+                <dd className={`status--badge status--badge__${data.state.toUpperCase()}`}>{displayCase(data.state)}</dd>
```

Comment:
Curious why this is `toUpperCase` instead of `toLowerCase`. Currently when the rule is disabled, the badge is still green.

Reply:
- Sorry for any confusion. I have designed all badges to be upper case based off of our EED EUI components (https://cdn.earthdata.nasa.gov/eui/latest/docs/eui/index.html#elements). 
- Oops. I left the test in there. Let me get that. Thanks for catching that.

Reply:
Does this apply to the class name itself though? Now I'm not seeing a background color for either state due to case sensitivity of the class selector.

Reply:
Oh I see what you are saying. Nevermind. Silly me. I was thinking about the badge itself and not className itself.

Reply:
Cool no worries, looks great! Thanks for the changes